packages/concerto-core/lib/introspect/stringvalidator.js

```diff
@@ -60,14 +89,38 @@ class StringValidator extends Validator{
      */
     validate(identifier, value) {
         if(value !== null) {
-            if(!this.regex.test(value)) {
-                this.reportError(identifier, 'Value \'' + value + '\' failed to match validation regex: ' + this.regex);
+            if (this.regex) {
```

Comment:
I think its better to avoid unnecessary nesting of "if" statements here. You could always merge this to one conditional check which would improve code readability